code/modules/antagonists/cult/cult_items.dm

```diff
@@ -563,6 +688,73 @@ Striking a noncultist, however, will tear their flesh."}
 
 #undef MAX_SHUTTLE_CURSES
 
+#define GATEWAY_TURF_SCAN_RANGE 40
+
+/obj/item/proteon_orb
+	name = "summoning orb"
+	desc = "An eerie translucent orb that feels impossibly light. Legends say summoning orbs are created from corrupted scrying orbs. If you hold it close to your ears, you can hear the screams of the damned."
+	icon = 'icons/obj/antags/cult/items.dmi'
+	icon_state = "summoning_orb"
+	light_range = 3
+	light_color = "#ff0000"
+
+/obj/item/proteon_orb/examine(mob/user)
+	. = ..()
+	if(!IS_CULTIST(user) && isliving(user))
+		var/mob/living/luser = user
+		luser.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
+		. += span_danger("It hurts just to look at it. Better keep away.")
+	else
+		. += span_cult("It can be used to create a gateway to Nar'Sie's domain, which will summon weak, sentient constructs over time.")
+
+/obj/item/proteon_orb/attack_self(mob/living/user)
+
+	var/list/turfs_to_scan = detect_room(get_turf(user), max_size = GATEWAY_TURF_SCAN_RANGE)
+
+	if(!IS_CULTIST(user))
+		to_chat(user, span_cult_large("\"You want to enter my domain? Go ahead.\""))
+		turfs_to_scan = null // narsie wants to have some fun and the veil wont stop her
+
+	for(var/turf/hole_candidate as anything in turfs_to_scan)
+		if(locate(/obj/structure/spawner/sentient/proteon_spawner) in hole_candidate)
+			to_chat(user, span_cult_bold("There's a gateway too close nearby. The veil is not yet weak enough to allow such close rips in its fabric."))
+			return
+	to_chat(user, span_cult_bold_italic("You focus on [src] and direct it into the ground. It rumbles..."))
+
+	var/turf/open/hole_spot = get_turf(user)
+	if(!istype(hole_spot) || istype(hole_spot, /turf/open/space))
+		to_chat(user, span_notice("This is not a suitable spot."))
+		return
+
+	INVOKE_ASYNC(hole_spot, TYPE_PROC_REF(/turf/open, quake_gateway), user)
+	qdel(src)
+
+/turf/open/proc/quake_gateway(mob/living/user)
```

Comment:
Open turf level proc with no autodoc

Reply:
The way you said that implies I should move the proc somewher else, but the orb is qdelled and the gateway hasnt spawned yet

Reply:
Just needs a doc